clients/privacy-center/packages/fides-consent/rollup.config.js

```diff
@@ -24,7 +24,7 @@ const generateConsentConfig = () => {
    */
   const privacyCenterConfig = require("../../config/config.json");
   const privacyCenterOptions =
-    privacyCenterConfig.consent?.consentOptions ?? [];
+    privacyCenterConfig.consent?.page.consentOptions ?? [];
```

Comment:
Hey @NevilleS, have made some changes to the overall readability of this config as prev. we ended up with too many different `title` and `description` objects.

As part of it I ended up in this file — is this the correct place to be changing this code?